### ISSUE TITLE
[Bug] Tracking Content dialog has unlabeled back button

### ISSUE THREAD

**Issue by asadotzler** [mozilla-mobile/fenix#8288]
## Steps to reproduce
Enable Talkback.
Focus the shield icon
Double-click to activate
Focus Tracking Content
Double click to activate

### Expected behavior
Back button in dialog has no label

### Actual behavior
Should have label

### Device information

* Android device: galaxy s10e
* Fenix version: 2/7/2020

**Comment by asadotzler** [comment]
Did this screen go away?

**Comment by AndiAJ** [comment]
Hi, **verified as fixed** on the latest **`Nightly Build #20860605 from 3/26`** using the following devices:
• Google Pixel 3a (Android 10)
• Huawei Mate 20 Lite (Android 8.1.0)
• OnePlus A3 (Android 6.0.1)
